.gitignore

```diff
@@ -1,5 +1,2 @@
-test-db/
-test-db-*/
 coverage
-node_modules/
-*.log
```

Comment:
Yes, it is, but not everyone has that.
So, we have two options here like I see it:
- update the CONTRIBUTORS.md with your suggestion
- adding those files to the .gitignore

`*.log` is e.g. the error log for npm